script.js

```diff
@@ -1,5 +1,87 @@
+import Player from './Player.js';
+import Log from './Log.js';
+
+let log = new Log;
+let hero = new Player('hero');
+let monster = new Player('monster', 150);
+
 let vue = new Vue({
-    el: '',
-    data: {},
-    computed: {}
+    el: '[data-behaviour=container]',
+    data: {
```

Comment:
For all constants. make a separate js file as constants from where my static values will be served.
player names, attackPowers, specialAmp